src/components/intro/index.tsx

```diff
@@ -1,26 +1,134 @@
     const fileInput = event.target as HTMLInputElement;
     const file = fileInput.files && fileInput.files[0];
     if (!file) return;
     this.props.onFile(file);
   }
 
-  render({ }: Props, { }: State) {
+  @bind
+  private onButtonClick() {
+    this.fileInput!.click();
+  }
+
+  @bind
+  private async onDemoClick(index: number, event: Event) {
+    try {
+      this.setState({ fetchingDemoIndex: index });
+      const demo = demos[index];
+      const blob = await fetch(demo.url).then(r => r.blob());
+
+      // Firefox doesn't like content types like 'image/png; charset=UTF-8', which Webpack's dev
+      // server returns. https://bugzilla.mozilla.org/show_bug.cgi?id=1497925.
+      const type = /[^;]*/.exec(blob.type)![0];
+      const file = new Fileish([blob], demo.filename, { type });
+      this.props.onFile(file);
+    } catch (err) {
+      this.setState({ fetchingDemoIndex: undefined });
+      this.props.onError("Couldn't fetch demo image");
+    }
+  }
+
+  render({ }: Props, { fetchingDemoIndex }: State) {
     return (
-      <div class={style.welcome}>
-        <h1>Drop, paste or select an image</h1>
-        <input type="file" onChange={this.onFileChange} />
+      <div class={style.intro}>
+        <div>
+          <div class={style.logoSizer}>
+            <div class={style.logoContainer}>
+              <img src={logo} class={style.logo} alt="Squoosh" />
+            </div>
+          </div>
+          <p class={style.openImageGuide}>
+            Drag &amp; drop or{' '}
+            <button class={style.selectButton} onClick={this.onButtonClick}>select an image</button>
+            <input
+              class={style.hide}
+              ref={linkRef(this, 'fileInput')}
+              type="file"
+              onChange={this.onFileChange}
+            />
+          </p>
+          <p>Or try one of these:</p>
+          <ul class={style.demos}>
+            {demos.map((demo, i) =>
+              <li key={demo.url} class={style.demoItem}>
+                <button class={style.demoButton} onClick={this.onDemoClick.bind(this, i)}>
+                  <div class={style.demo}>
+                    <div class={style.demoImgContainer}>
+                      <div class={style.demoImgAspect}>
+                        <img class={style.demoIcon} src={demo.iconUrl} alt=""/>
+                        {fetchingDemoIndex === i &&
+                          <div class={style.demoLoading}>
+                            <loading-spinner class={style.demoLoadingSpinner}/>
+                          </div>
+                        }
+                      </div>
+                    </div>
+                    <div class={style.demoDescription}>{demo.description}</div>
+                  </div>
+                </button>
+              </li>,
+            )}
+          </ul>
+        </div>
+        <ul class={style.relatedLinks}>
+          <li><a href="https://github.com/GoogleChromeLabs/squoosh/">View the code</a></li>
```

Comment:
Think it'd be worth going `target=_blank` here?

Reply:
I'm never quite sure what the right thing to do here is. I usually leave it to the user to decide how links should open.

https://carbon.now.sh
https://meowni.ca/font-style-matcher/
https://yoksel.github.io/url-encoder/
https://davidshimjs.github.io/qrcodejs/

None of these use `target=_blank`. Know any examples that do?

Reply:
carbon is probably the right example here, so maybe we just run without. I tend to use a new tab when the app has unrecoverable state and the link is explanatory (eg: footer link), but its probably just preference.

Reply:
> I tend to use a new tab when the app has unrecoverable state

That's a fair point. Currently the link only appears on the initial screen so there's no state, but I'd like to think about it again if we put the link somewhere where we have state.